pages/activities/[id]/index.js

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from "next/router";
+import ActivityDetails from "@/components/ActivityDetails";
+import Link from "next/link";
+import Navigation from "@/components/Navigation";
```

Comment:
Instead of importing and rendering the Navigation in different files you should define a global layout in the _app.js.